src/Build/BackEnd/Components/SdkResolution/SdkResolverLoader.cs

```diff
@@ -23,8 +23,9 @@ internal class SdkResolverLoader
         internal virtual IList<SdkResolver> LoadResolvers(LoggingContext loggingContext,
             ElementLocation location)
         {
-            // Always add the default resolver
-            var resolvers = new List<SdkResolver> {new DefaultSdkResolver()};
+            var resolvers = Environment.GetEnvironmentVariable("MSBUILDINCLUDEDEFAULTSDKRESOLVER")?.ToLowerInvariant() != "false" ? 
+                new List<SdkResolver> {new DefaultSdkResolver()}
+                : new List<SdkResolver>();
```

Comment:
I'm now realizing this does something different than what I originally thought.  It disables the default SDK resolver, which is different than disabling loading resolvers from the default Sdk folder.  I can't remember what you said you added this for, are you sure it is going to do what you meant?

Reply:
I included this so that I could ensure SDK resolution fails when we can't find the global.json specified version (before my change). When I was testing this test hook along with my SDK tests ([branch here](https://github.com/dotnet/sdk/compare/master...sfoslund:ResolverTests?expand=1)), SDK resolution would fail with the MSBuildSDK resolver so it would fall back to the default resolver which would actually succeed. I wanted to have a test case to ensure that outside of VS SDK resolution really does fail, which requires disabling the default SDK.